nemo/backends/pytorch/actions.py

```diff
@@ -537,6 +537,7 @@ def _eval(self, tensors_2_evaluate, callback, step, verbose=False):
                         'num_workers': dl_nm.num_workers,
                         'batch_size': dl_nm.batch_size,
                         'shuffle': False,
+                        'pin_memory': dl_nm.pin_memory,
```

Comment:
Interesting... Are you moving data to GPU already in SGD datalayer?

Reply:
hey @okuchaiev check it out, yet another parameter that we should add to **every** DataLayer... ;)